scripts/create-clog.sh

```diff
@@ -99,8 +99,16 @@ function update_manifest () {
   popd > /dev/null
 }
 
+function print_pr_info () {
+  branch="$(git branch --show-current)"
+  echo "tip: After comitting this branch visit"
```

Comment:
```suggestion
  echo "tip: After committing this branch visit"
```

Code Spell Checker is useful for picking up typos in VSCode :)

Reply:
I actually use Code Spell Checker, just didn't have it enabled on `sh` files

Reply:
Oh, I have it on for everything I think :)

Reply:
```json
"cSpell.enabledLanguageIds": [
    "json",
    "jsonc",
    "markdown",
    "plaintext",
    "restructuredtext",
    "text",
    "yaml",
    "yml"
  ],
```

I think I did this because there were blue squigglies everywhere. I'll have to revisit and maybe just start adding certain words to the workspace dict

Reply:
haha yea I have just been ignoring those squigglies but adding stuff to a dictionary and checking it in is definitely useful!

also this one I just caught because I'm neurotic :)